src/styles/_variables.scss

```diff
@@ -1,14 +1,24 @@
 @import "https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700";
+@import 'https://fonts.googleapis.com/css2?family=Norican&display=swap';
 
 $mainFont: "Montserrat", sans-serif;
-
+$quoteFont: "Norican", cursive;
+ 
+$text: #000000; 
+$secondaryColor : #180C30;
+$purpleHeading: #581184;
+$lightPinkBackground: #F0EAF4; 
+$mainColor: #FFFFFF;
+$deleteColor: #C91212;
```

Comment:
If you create these variables in `_variables.scss` file, then should be use color variables for each value from `_colors.scss`, also would be better move all color variables to `_colors.scss` file